src/pages/StyleGuide.js

```diff
@@ -65,15 +64,8 @@ class StyleGuide extends Component {
       ]),
       div({ style: styles.container }, [
         validatedInput({
-          name: 'input',
-          inputProps: {
-            placeholder: 'ValidatedInput wants an email',
-            value: this.state.validatedInputValue,
-            onChange: e => this.setState({ validatedInputValue: e.target.value, validatedInputTouched: true })
-          },
-          errors: this.state.validatedInputTouched ?
-            validate.single(this.state.validatedInputValue, { email: true }) :
-            null
+          inputProps: { value: 'Hello there!', readOnly: true },
```

Comment:
Can we make this writable? One of the things that is useful here is to be able to see the validation behavior.